docs/istio-operator-overview.svg

```diff
@@ -0,0 +1,4 @@
bject><text x="1684" y="219" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">Kyma Gateway</text></switch></g><rect x="28.5" y="730" width="310" height="130" fill="none" stroke="rgb(0, 0, 0)" pointer-events="all"/><path d="M 238.5 745 L 288.5 745 L 318.5 775 L 318.5 845 L 238.5 845 L 238.5 745 Z" fill="#ffe6cc" stroke="#d79b00" stroke-miterlimit="10" stroke-dasharray="1 1" pointer-events="all"/><path d="M 288.5 745 L 288.5 775 L 318.5 775 Z" fill-opacity="0.05" fill="#000000" stroke="none" pointer-events="all"/><path d="M 288.5 745 L 288.5 775 L 318.5 775" fill="none" stroke="#d79b00" stroke-miterlimit="10" stroke-dasharray="1 1" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 78px; height: 1px; padding-top: 795px; margin-left: 240px;"><div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;"><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">kyma resources<br />istio</div></div></div></foreignObject><text x="279" y="799" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">kyma resource...</text></switch></g><rect x="685" y="10" width="560" height="50" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe flex-start; justify-content: unsafe flex-start; width: 552px; height: 1px; padding-top: 0px; margin-left: 690px;"><div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: left; max-height: 60px; overflow: hidden;"><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;"><h1>IstioOperator overview and resource ownership</h1></div></div></div></foreignObject><text x="690" y="12" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px">IstioOperator overview and resource ownership</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Text is not SVG - cannot display</text></a></switch></svg>
```

Comment:
The `configured by` connection between Istio CR, Kyma Istio Operator defaults and IstioController suggests, that those resources are configured by the Istio Controller, not that it uses them as configuration for controller runtime

Reply:
You are right, that is confusing, because the arrows would need to point in the opposite direction. Do you have any proposal? How about "triggers reconciliation" for the IstioCR and "consumes/uses" for the Operator defaults? Also the arrow direction must be changed depending on the wording.

Reply:
I would also argue that the `reconciles` connection for Istio Installation could be replaced with `triggers`, as the main reconciler doesn't configure any state based on the Istio installation